assets/stylesheets/application.scss

```diff
@@ -1,48 +1,8 @@
-@import 'tailwindcss/base';
-@import 'tailwindcss/components';
-@import 'tailwindcss/utilities';
+@import 'dependencies';
```

Comment:
How about naming this `vendor` similarly to Rails?

Reply:
Updated here https://github.com/Lahphim/go-crawler-challenge/pull/38/commits/c78ebb097ac24e031c915288ebe22b0343590d77